src/pybind/mgr/dashboard/run-frontend-e2e-tests.sh

```diff
@@ -50,6 +50,7 @@ stop() {
         for cluster in ${CLUSTERS[@]}; do
             ../src/mstop.sh $cluster
         done
+        killall rbd-mirror
```

Comment:
Just to be certain in case the daemon is locked up for some reason:
```suggestion
        killall -9 rbd-mirror
```

Reply:
done, thanks.